### DISCUSSION TITLE
paddlespeech asr --lang en 的问题

### DISCUSSION THREAD

**Discussion by josephthlee** [PaddlePaddle/PaddleSpeech#2445] [Q&A] [answered]
关于en ASR，请教一个问题，我看到教程

![18261663981255_ pic](https://user-images.githubusercontent.com/30864375/192073589-db1d14c8-ace8-437e-bcaf-98240629bdea.jpg)

![18271663981282_ pic](https://user-images.githubusercontent.com/30864375/192073596-194f38be-2c6e-416d-a672-97e7e1f5f728.jpg)

然后我尝试 去 PaddleSpeech ASR en 语言 

```
 paddlespeech asr --lang en  --input en.wav 
```

结果报错，

```
AssertionError: Can't find "conformer_wenetspeech-en-16k" in resource. Model name must be one of ['conformer_wenetspeech-zh-16k', 'conformer_online_wenetspeech-zh-16k', 'conformer_online_multicn-zh-16k', 'conformer_aishell-zh-16k', 'conformer_online_aishell-zh-16k', 'transformer_librispeech-en-16k', 'deepspeech2online_wenetspeech-zh-16k', 'deepspeech2offline_aishell-zh-16k', 'deepspeech2online_aishell-zh-16k', 'deepspeech2offline_librispeech-en-16k'](base) root@UNKNOWN:/home/misterbigbook/Workspace/paddle/PaddleSpeech/sample# 
```

请问下 ASR 英文audio 是要怎么处理？我是不是缺了什么呢？
谢谢！

**Comment by yt605155624** [comment, marked as answer]
参考 https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/demos/speech_recognition 需要用 `--model` 选择对应的英文模型

**Reply by josephthlee** [reply]
感谢，已测试没问题！
